proposals/utils/pdf_diff_logic.py

```diff
@@ -1037,7 +1032,6 @@ def create_context_pdf(context, model):
                     if study.has_observation:
                         sections.append(ObservationSection(study.observation))
```

Comment:
I think all the major sections like `ObservationSection` and `InterventionSection` should get the `PageBreakMixin` treatment, and probably also each task and session.

If the subtitles like "Sesion 1, task 1" could also get a decent font size bump that would also be a great improvement.